packages/ui-concerto/src/lib/reactformvisitor.js

```diff
@@ -46,6 +44,8 @@ import {
   toFieldType,
 } from './utilities';
 
+const toPath = (paramsArr) => paramsArr.join('.');
+
 /**
  * Returns true if the field has the @FormEditor( "hide", true )
```

Comment:
Did you re-test with hidden fields, to ensure we don't get `null` children?

Reply:
We will have a `null` child if a field is hidden, but that's the correct approach: https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering.